gulp/tasks/test.js

```diff
@@ -93,6 +94,36 @@ function _getProtractorParams() {
   return params;
 }
 
+gulp.task( 'test:acceptance:perf', sitespeedio( {
+  url: 'http://localhost:8000/',
```

Comment:
Pull in values from .env. See https://github.com/cfpb/cfgov-refresh/blob/flapjack/gulp/tasks/test.js#L117-L118 You will need the ESLint ignores till we move those all to their own module. Also, we should drop the `||` values and just make the host and port turned on by default in `.env`, but you can leave them in there for now.